source/network/connecting-palos.html.md.erb

```diff
@@ -73,4 +73,6 @@ You can connect to the Palo VMs via the HMCTS VPN by following the following ste
 - [hmcts-hub-prod-int-palo-vm-0](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/0978315c-75fe-4ada-9d11-1eb5e0e0b214/resourceGroups/hmcts-hub-prod-int/providers/Microsoft.Compute/virtualMachines/hmcts-hub-prod-int-palo-vm-0/overview)
 - [hmcts-hub-prod-int-palo-vm-1](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/0978315c-75fe-4ada-9d11-1eb5e0e0b214/resourceGroups/hmcts-hub-prod-int/providers/Microsoft.Compute/virtualMachines/hmcts-hub-prod-int-palo-vm-1/overview)
 
+#### SSH to PaloAlto VMs
```

Comment:
what credentials am I looking for in that KV?

Reply:
Updated to be more specific on the secret names within the KV